aiogram/api/types/chat_member.py

```diff
@@ -65,3 +65,9 @@ class ChatMember(TelegramObject):
     inline bots"""
     can_add_web_page_previews: Optional[bool] = None
     """Restricted only. True, if the user is allowed to add web page previews to their messages"""
+
+    def is_chat_admin(self) -> bool:
+        return self.status in {"creator", "administrator"}
```

Comment:
what if we make it `property`?
by the way, we need to ensure `self.status` has the right case (I think, using `str.lower` would be enough. However there's `str.casefold` for both operands)

Reply:
I think, `status` values should be enumerated.

Reply:
@uwinx, is not needed. API can generate this values only in lowercase